--- gnotify-1.0/src/Makefile.am	2004-05-20 18:55:26.000000000 +0200
+++ gnotify-1.0.new/src/Makefile.am	2004-06-01 21:18:07.758890344 +0200
@@ -9,8 +9,8 @@
 
 AM_CFLAGS =\
 	`pkg-config --cflags libxml-2.0`\
-	 -Wall\
-	 -O3
+	 -Wall
+#	 -O3
 
 bin_PROGRAMS = gnotify
 
--- gnotify-1.2/configure.in.orig	2004-08-31 13:35:18.000000000 +0200
+++ gnotify-1.2/configure.in	2025-08-31 19:54:24.286509679 +0200
@@ -8,8 +8,6 @@ AM_INIT_AUTOMAKE(GNotify, 1.2)
 AM_CONFIG_HEADER(config.h)
 
 AC_ISC_POSIX
-CFLAGS=""
-AC_SUBST(CFLAGS,$CFLAGS)
 AC_PROG_CC
 AM_PROG_CC_STDC
 AC_HEADER_STDC
